src/firefly/js/tables/reducer/TableDataReduer.js

```diff
@@ -18,15 +20,18 @@ export function dataReducer(state={data:{}}, action={}) {
                 return update(root, { [tbl_id] : {selectInfo: {$set: selectInfo}}});
             } else return root;
 
+        case (Cntlr.TABLE_NEW_LOADED)  :
+            if (get(root, [tbl_id, 'tableMeta', 'Loading-Status'], 'COMPLETED') !== 'COMPLETED') {
+                return update(root, { [tbl_id] : {tableMeta: {['Loading-Status']:  {$set: 'COMPLETED'}}}});
```

Comment:
could this be?
 `return update(root, { [tbl_id] : {tableMeta: {Loading-Status:  {$set: 'COMPLETED'}}}});`

As I said earlier,  I am begging to really not like update because it is so hard to read. I though you wrote something else to stick an object into a deep object?